android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -605,7 +606,7 @@ private void startPlayback() {
             initializePlayer();
         }
         if (!disableFocus) {
-            setKeepScreenOn(true);
+            setKeepScreenOn(true && preventsDisplaySleepDuringVideoPlayback);
```

Comment:
You can remove this `true` here as it doesn't do anything :)

Reply:
Done ✅